JeMPI_Apps/JeMPI_API/pom.xml

```diff
@@ -194,6 +194,11 @@
             <artifactId>akka-actor-testkit-typed_${scala.tools.version}</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
There's another LibMPI dependency at line 27, I don't get the difference, is it a duplicate ?